504.xml

```diff
@@ -0,0 +1,362 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- 
+FILE INFORMATION
+OMA Permanent Document
+   File: OMA-SUP-XML_504-V1_0-YYYY-MM-DD-A
+   Path: https://www.openmobilealliance.org/release/ObjLwM2M_Remote_SIM_Provisioning
+   Type: xml
```

Comment:
Following metadata to be removed as it's no longer required
  Type: xml
   Date: YYYY-MMM-DD